src/main/resources/liquibase.properties

```diff
@@ -4,7 +4,7 @@ changeLogFile=src/main/resources/db/changelog/db.changelog-master.xml
 # The target change log file to output to.
 # If this is null then the output will be to the screen.
 #outputChangeLogFile=src/main/resources/db/changelog/liquibase-outputChangeLog.xml
-url=jdbc:postgresql://localhost:5050/ccd_data
-username=ccd
-password=ccd
+url=jdbc:postgresql://localhost:5451/role_assignment
+username=am
+password=am
 driver=org.postgresql.Driver
```

Comment:
please check in the .env file as well. It will help to keep track of the required environment variable for bootrun mode.

Reply:
Done